README.rst

```diff
@@ -10,10 +10,13 @@ Vagrant
 =======
 
 `Vagrant <https://www.vagrantup.com/>`_ allows contributors to get quickly up and running with a
-Bodhi development environment by automatically configuring a virtual machine. To get started, simply
+Bodhi development environment by automatically configuring a virtual machine. Before you get
+started, ensure that your host machine has virtualization extensions enabled in its BIOS so the
+guest doesn't go slower than molasses. To get started, simply
 use these commands::
 
-    $ sudo dnf install ansible vagrant-libvirt vagrant-sshfs
+    $ sudo dnf install ansible libvirt vagrant-libvirt vagrant-sshfs
+    $ for a in {enable,start}; do sudo systemctl $a libvirtd; done;
```

Comment:
I like that it's one line to copy/paste instead of two, but I'll fix it as per your preference.